frontend/src/components/annotation/audio_annotation/audio_annotation_core.vue

```diff
@@ -126,7 +81,19 @@ export default {
     label_schema: {
       type: Object,
       default: {}
-    }
+    },
```

Comment:
Passing new properties to `audio_annotation_core` that are passed by `annotatino_ui_factory` (parent component)

`instance_store` is used to share internal `instance_list` state between child (`auido_annotation_core`) and ancestor (`annotation_ui_factory`) components

`annotation_ui_context` stores  `current_label_file` (currently selected label), `command_manager`

`working_file` stores information about currently annotated file. Need `working_file.id` to enable saving `instance_list` for the task context.